src/foam/nanos/theme/services.jrl

```diff
@@ -63,7 +63,7 @@ p({
   "serviceScript": """
     return new foam.dao.EasyDAO.Builder(x)
       .setOf(foam.nanos.theme.ThemeDomain.getOwnClassInfo())
-      .setAuthorizer(new foam.nanos.auth.GlobalReadAuthorizer("themeDomainDAO"))
+      .setAuthorizer(new foam.nanos.auth.GlobalReadAuthorizer("themeDomain"))
```

Comment:
```suggestion
      .setAuthorizer(new foam.nanos.auth.GlobalReadAuthorizer("themedomain"))
```

Reply:
Why do you want to change the case here? (and not the others)
They should all be changed, but I was leaving that for another PR.

Reply:
this is the only one where we're explicitly granting the permission to some group using all lowercase